src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerFunctionalTest.java

```diff
@@ -249,7 +249,7 @@ public void should_succeed_and_create_a_task_with_single_categories() {
             .extract()
             .path("[0].id");
 
-        taskIdStatusMap.put(caseId1Task1Id, "completed");
+        //taskIdStatusMap.put(caseId1Task1Id, "completed");
```

Comment:
Can just delete.  Github will tell you what changed.

Reply:
Yes Ravi, Removed all the commented code.